### DISCUSSION TITLE
Can't redeem codes :/

### DISCUSSION THREAD

**Discussion by benny98tr** [BennyGaming635/My-C--Projects#5] [Q&A] [answered]
Trying to use code 'WELCOMETOTHEGAME' and I'm getting an error. The code apparently dosen't exist...

**Comment by BennyGaming635** [comment, marked as answer]
Hey @benny98tr,

Nice to see you again. Note that the code has to be in uppercase to work. Lowercase won't work. If that still isn't working note that i've now pushed an update to the game to hopefully fix your issue if thats not working! :octocat:

**Reply by benny98tr** [reply]
Sweet! I made a mistake when doing the code.

Thanks!
